test/unit/org/apache/cassandra/net/ConnectionTest.java

```diff
@@ -487,13 +487,13 @@ public long serializedSize(Object o, int version)
                            .overload (  0,  0)
                            .expired  (  0,  0)
                            .error    ( 10, 10 * message.serializedSize(version))
-                           .check();
+                           .longCheck(  5, SECONDS);
```

Comment:
would also be good to document why not check; you say it in the JIRA but a code comment is easier to future readers.